src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -38,13 +38,19 @@ namespace AppInstaller::CLI::Workflow
 
     namespace anon
     {
+        static const AppInstaller::Utility::Version s_MinimumSchemaVersionModuleNameRequiredInType = { "0.3" };
+
         constexpr std::wstring_view s_Directive_Description = L"description";
         constexpr std::wstring_view s_Directive_Module = L"module";
         constexpr std::wstring_view s_Directive_AllowPrerelease = L"allowPrerelease";
 
         constexpr std::wstring_view s_Unit_WinGetPackage = L"WinGetPackage";
         constexpr std::wstring_view s_Unit_WinGetSource = L"WinGetSource";
 
+        constexpr std::wstring_view s_UnitType_WinGetPackage_DSCv3 = L"Microsoft.WinGet/WinGetPackage";
+        constexpr std::wstring_view s_UnitType_WinGetSource_DSCv3 = L"Microsoft.WinGet/WinGetSource";

```

Comment:
```suggestion
        constexpr std::wstring_view s_UnitType_WinGetPackage_DSCv3 = L"Microsoft.WinGet/Package";
        constexpr std::wstring_view s_UnitType_WinGetSource_DSCv3 = L"Microsoft.WinGet/Source";
```
Also, to prevent conflicts I'm making the dev build use `Microsoft.WinGet.Dev" as the root name.  I can deal with the fallout though.

Reply:
Do we need to add UserSettingsFile resource here?

Reply:
It's declared in below PredefinedResources